crt_portal/cts_forms/migrations/0087_standardize_form_letter_subjects.py

```diff
@@ -0,0 +1,29 @@
+# Generated by Django 2.2.13 on 2020-08-17 18:07
+
+from django.db import migrations
+
+
+def tweak_subject_lines(apps, schema_editor):
+    # add "the" as a prefix
+    common_subject = "Response: Your Civil Rights Division Report - {{ record_locator }} from the {{ section_name }} Section"
+    templates = ResponseTemplate.objects.filter(subject__icontains='from {{ section_name }}')
```

Comment:
Ooo, we need to import `ResponseTemplate` from somewhere 😃

Reply:
:see_no_evil: oh boy. my mistake completely.